.github/workflows/tests_lgpumpi_python.yml

```diff
@@ -27,7 +27,7 @@ env:
   CI_CUDA_ARCH: 86
 
 concurrency:
-  group: tests_lgpumpi_python-${{ github.ref }}-${{ inputs.lightning-version }}-${{ inputs.pennylane-version }}
+  group: tests_lgpumpi_python-${{ github.ref }}-${{ github.event }}-${{ inputs.lightning-version }}-${{ inputs.pennylane-version }}
```

Comment:
I'm quite curious about what could happen if we remove this constrain here.

Reply:
Workflows triggered by different events could cancel one another, which I wanted to avoid.